plenum/server/consensus/consensus_shared_data.py

```diff
@@ -41,6 +41,8 @@ def __init__(self, name: str, validators: List[str], inst_id: int):
         self.node_mode = Mode.starting
         # ToDo: it should be set in view_change_service before view_change starting
         self.legacy_last_prepared_before_view_change = None
+        self.primaries_batch_needed = False
+        self.requestQueues = {}
```

Comment:
I think `requestQueues ` needs to be internal` OrderingService`'s field. `OrderingServic`e needs to subscribe to `ReqKey` events via Internal Bus and add reqs to `requestQueues `